src/lib.rs

```diff
@@ -14,6 +14,7 @@ extern crate lzw;
 extern crate num_iter;
 extern crate num_rational;
 extern crate num_traits;
+extern crate scoped_threadpool;
```

Comment:
This dependency is included only for `hdr` and wouldn't work in `src/lib.rs`. Probably CI will flag this as well.